fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -433,8 +445,8 @@ public void delete() {
 
             node.remove();
 
-            if (parent != null) {
-                createTombstone(parent, name);
+            if (!isTimeMap && parent != null) {
```

Comment:
An inline comment here would be helpful, explaining why creating a tombstone is not done for timemaps and resources without parents.